staging/src/k8s.io/legacy-cloud-providers/azure/azure_fakes.go

```diff
@@ -991,3 +998,54 @@ func (f *fakeVMSet) GetDataDisks(nodeName types.NodeName, crt azcache.AzureCache
 func (f *fakeVMSet) GetPowerStatusByNodeName(name string) (string, error) {
 	return "", fmt.Errorf("unimplemented")
 }
+
```

Comment:
maybe could add a comment here: why make this func public

Reply:
done